app/controllers/measurements_controller.rb

```diff
@@ -56,6 +56,40 @@ def new_meteorological
     end
   end
 
+  def new_wind_turbine_speed
```

Comment:
Speed is the only field of the WindTurbineSeedMeasurement model. Could we use only one model/table called WindTurbineMeasurement and include there all the fields related to the WindTurbine?